src/actions/schedule-actions.js

```diff
@@ -7,30 +7,77 @@ export const UPDATE_FILTER      = 'UPDATE_FILTER';
 export const UPDATE_FILTERS     = 'UPDATE_FILTERS';
 export const CHANGE_VIEW        = 'CHANGE_VIEW';
 export const CHANGE_TIMEZONE    = 'CHANGE_TIMEZONE';
-
+/**
+ * This action is defined to just reinitialize the allScheduleReducer state
+ * (allSchedulesReducer.state.schedules) without trigerring the side effects of
+ * SYNC_DATA that reloads all the reducers to its initial state
+ * @type {string}
+ */
+export const RELOAD_SCHED_DATA = 'RELOAD_SCHED_DATA';
+/**
+ * this action is defined to reload the user schedules after the schedule data is
+ * initialized (allSchedulesReducer.state.schedules.length > 0 )
+ * @type {string}
+ */
+export const RELOAD_USER_PROFILE = 'RELOAD_USER_PROFILE';
 
 const fragmentParser = new FragmentParser();
 
 export const updateFilter = (key, filter, action = UPDATE_FILTER) => (dispatch) => {
     dispatch(createAction(action)({...filter, key}))
 };
 
-export const updateFiltersFromHash = (key, filters, view, actionCallback = UPDATE_FILTERS) => (dispatch) => {
-    const qsFilters = fragmentParser.getParams();
 
-    // clear hash
-    fragmentParser.clearParams();
+export const deepLinkToEvent = () => {
```

Comment:
Should this be scoped as an action? Since it only resolves the presentational aspect.